test/langtools/jdk/javadoc/doclet/testHtmlVersion/TestHtmlVersion.java

```diff
@@ -142,25 +142,6 @@ void html5Output() {
                 + "<nav role=\"navigation\">\n"
                 + "<!-- ======= START OF BOTTOM NAVBAR ====== -->");
 
-        // Test for package-frame page
-        checkOutput("pkg/package-frame.html", true,
-                "<!DOCTYPE HTML>",
-                "<meta name=\"dc.created\"",
-                "<main role=\"main\">\n"
-                + "<h1 class=\"bar\"><a href=\"package-summary.html\" target=\"classFrame\">pkg</a></h1>",
-                "<section role=\"region\">\n"
-                + "<h2 title=\"Interfaces\">Interfaces</h2>",
-                "<section role=\"region\">\n"
-                + "<h2 title=\"Classes\">Classes</h2>",
-                "<section role=\"region\">\n"
-                + "<h2 title=\"Enums\">Enums</h2>",
-                "<section role=\"region\">\n"
-                + "<h2 title=\"Exceptions\">Exceptions</h2>",
-                "<section role=\"region\">\n"
-                + "<h2 title=\"Errors\">Errors</h2>",
-                "<section role=\"region\">\n"
-                + "<h2 title=\"Annotation Types\">Annotation Types</h2>");
-
```

Comment:
I recently backported the original change JDK-8221871 to some other release and noticed this diff. I wonder why these lines were deleted? They seem related to JDK-8215599 which hasn't been backported.

Reply:
Hi, while backporting JDK-8221871 I've seen test failures at this code hence removed it which passed the tests. I wasn't aware JDK-8215599 needs to be backported first.

Reply:
I cannot see that JDK-8215599 has been ever backported, which is fine; I didn't say it should. However, I somehow managed to make those deleted lines work by accounting for the change brought by JDK-8221871 and was simply wondering what happened here. Here's the ammended change:
```
        // Test for package-frame page
        checkOutput("pkg/package-frame.html", true,
                "<!DOCTYPE HTML>",
                "<meta name=\"dc.created\"",
                "<main role=\"main\">\n"
                + "<h1 class=\"bar\"><a href=\"package-summary.html\" target=\"classFrame\">pkg</a></h1>",
                "<section>\n"
                + "<h2 title=\"Interfaces\">Interfaces</h2>",
                "<section>\n"
                + "<h2 title=\"Classes\">Classes</h2>",
                "<section>\n"
                + "<h2 title=\"Enums\">Enums</h2>",
                "<section>\n"
                + "<h2 title=\"Exceptions\">Exceptions</h2>",
                "<section>\n"
                + "<h2 title=\"Errors\">Errors</h2>",
                "<section>\n"
                + "<h2 title=\"Annotation Types\">Annotation Types</h2>");
```

Reply:
> Hi, while backporting JDK-8221871 I've seen test failures at this code hence removed it which passed the tests. I wasn't aware JDK-8215599 needs to be backported first.

Thanks for clarification.